src/main/com/smithsmodding/armory/client/textures/MaterializedTextureCreator.java

```diff
@@ -60,31 +55,7 @@
 
     //Initializes the dummy gui material with a proper set of render info.
     static {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)